nextflow.config

```diff
@@ -28,11 +28,12 @@ params {
     adapterremoval_adapter2              = 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTAGATCTCGGTGGTCGCCGTATCATT'
     adapterremoval_trim_quality_stretch  = false
     keep_phix                            = false
+    skip_shortread_qc                    = false
     // long read preprocessing options
     longread_adaptertrimming_tool        = "porechop_abi"
     longread_filtering_tool              = "filtlong"
     // phix_reference                    = "ftp://ftp.ncbi.nlm.nih.gov/genomes/genbank/viral/Enterobacteria_phage_phiX174_sensu_lato/all_assembly_versions/GCA_002596845.1_ASM259684v1/GCA_002596845.1_ASM259684v1_genomic.fna.gz"
-    phix_reference                       = "${baseDir}/assets/data/GCA_002596845.1_ASM259684v1_genomic.fna.gz"
+    phix_reference                       = null
```

Comment:
Why do you do this (also below)? Isn't it good to have defaults for these?

Reply:
Ah! This shouldn't have happened! Thank you for catching this!

Reply:
Isn't it quite risky to add paths to the code? (I just advised Danilo strongly against this...)